use-cases/virgo/config.yaml

```diff
@@ -17,6 +20,9 @@ training_pipeline:
           validation_proportion: 0.1
           rnd_seed: 42
           root_folder: ${root_folder}
+          hdf5_file_location: ${hdf5_file_location}
+          chunk_size: ${chunk_size}
+          hdf5_dataset_name: ${hdf5_dataset_name}
       - class_path: trainer.NoiseGeneratorTrainer
         init_args:
           config:
```

Comment:
this would become

```yaml
config:
  generator: simple #unet
  batch_size: 1
  optim_lr: ${learning_rate}
  loss: l1
  save_best: true
  shuffle_train: true
```